capstone/src/main/java/com/google/sps/servlets/ParseGroupDataServlet.java

```diff
@@ -0,0 +1,132 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Scanner;
+import java.util.Random;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.PrintWriter;
+import com.google.sps.Objects.User;
+import com.google.sps.Objects.Post;
+import com.google.sps.Objects.Comment;
+import com.google.sps.Objects.Group;
+import com.google.sps.Objects.response.PostResponse;
+import com.google.sps.servlets.ServletHelper;
+import error.ErrorHandler;
+
+/**
+ * Parses a TSV file of fake group data and populates the Datastore with Group and Post entities.
+ */
+@WebServlet("/parse-group-data")
+public class ParseGroupDataServlet extends HttpServlet {
```

Comment:
`ParseTestGroupDataServlet` and anything else regarding fake data should be obnoxiously pointing out this is for testing only